include/mata/util.hh

```diff
@@ -19,6 +19,7 @@
 #ifndef _MATA_UTIL_HH_
 #define _MATA_UTIL_HH_
 
+#include <bitset>
```

Comment:
why is this import here when nothing else changed in this file? Only empty lines were added.

Reply:
it was some experiment, will remove